format/spec.md

```diff
@@ -205,15 +205,40 @@ Supported primitive types are defined in the table below. Primitive types added
 |                  | **`uuid`**         | Universally unique identifiers                                           | Should use 16-byte fixed                         |
 |                  | **`fixed(L)`**     | Fixed-length byte array of length L                                      |                                                  |
 |                  | **`binary`**       | Arbitrary-length byte array                                              |                                                  |
+| [v3](#version-3) | **`geometry(C)`**  | Geometry features from [OGC – Simple feature access][1001]. Edge-interpolation is always linear/planar. See [Appendix G](#appendix-g-geospatial-notes). Parameterized by CRS C. If not specified, C is `OGC:CRS84`. |                                                        |
+| [v3](#version-3) | **`geography(C, A)`**  | Geometry features from [OGC – Simple feature access][1001]. See [Appendix G](#appendix-g-geospatial-notes). Parameterized by CRS C and edge-interpolation algoritm A. If not specified, C is `OGC:CRS84`. |                                                        |
```

Comment:
Just a minor clarification: since we are not specifying the default value for A, if the user provides a single argument geography specialization, it would be treated as `geography(A)`, correct?

Reply:
That's a good point, I think we can put 'spherical' as default?   cc @jiayuasu

Reply:
I think making 'spherical' a default is a good idea.

Reply:
Done.

Reply:
Thanks @szehon-ho!

Reply:
LGTM！